parachain/pallets/ethereum-beacon-client/src/benchmarking/mod.rs

```diff
@@ -42,16 +76,19 @@ benchmarks! {
 	import_execution_header {
 		let caller: T::AccountId = whitelisted_caller();
 
-		EthereumBeaconClient::<T>::initial_sync(data::initial_sync()).unwrap();
+		let initial_sync_data = initial_sync();
+
+		EthereumBeaconClient::<T>::initial_sync(initial_sync_data.clone())?;
 
-		let sync_update: SyncCommitteePeriodUpdate<T::MaxSignatureSize, T::MaxProofBranchSize, T::MaxSyncCommitteeSize> = data::sync_committee_update();
-		SyncCommittees::<T>::set(sync_update.sync_committee_period+1, sync_update.next_sync_committee);
+		let block_update = block_update();
 
-		let block_update = data::block_update();
+		SyncCommittees::<T>::insert(EthereumBeaconClient::<T>::compute_current_sync_period(
+				block_update.block.slot,
+			), initial_sync_data.current_sync_committee);
 
 		LatestFinalizedHeaderState::<T>::set(FinalizedHeaderState{
 			beacon_block_root: H256::default(),
-			beacon_slot: block_update.block.slot,
+			beacon_slot: block_update.block.slot - 1,
```

Comment:
Why do we need to add minus 1 here? 🤔

Reply:
yes, mistake here. for `import_execution_header` `LatestFinalizedHeaderState` should not be initialized with a parent slot of the update.